src/components/PageSidebar/SidebarTabAttachments.vue

```diff
@@ -272,14 +294,17 @@ export default {
 
 	watch: {
 		'page.id'() {
-			this.load('attachments')
-			this.getAttachmentsForPage()
+			this.getAttachmentsForPage(true)
```

Comment:
I think you need to set `loaded` to `false` here. Otherwise the offline indicator won't show even when navigating to pages whose attachments have never been loaded.

Reply:
Good catch as well. Fixed now.